lib/sdk/writer/index.js

```diff
@@ -24,10 +29,54 @@ const BlockWriteStream = require('./block-write-stream')
 const BlockReadStream = require('./block-read-stream')
 const ChecksumStream = require('./checksum-stream')
 const ProgressStream = require('./progress-stream')
+const imageStream = require('../image-stream')
+const diskpart = require('../../cli/diskpart')
+const constraints = require('../../shared/drive-constraints')
+const errors = require('../../shared/errors')
 const debug = require('debug')('etcher:writer')
-const EventEmitter = require('events').EventEmitter
 const _ = require('lodash')
 
+/* eslint-disable prefer-reflect */
+
+/**
+ * @summary Timeout, in milliseconds, to wait before unmounting on success
+ * @constant
+ * @type {Number}
+ */
+const UNMOUNT_ON_SUCCESS_TIMEOUT_MS = 2000
+
+/**
+ * @summary Helper function to run a set of async tasks in sequence
```

Comment:
I wonder if such a "generic mechanism" should be moved to a more "shared" location, rather than only being in `lib/sdk/writer/index.js` ?